core/model/modx/moduser.class.php

```diff
@@ -341,6 +341,9 @@ public function getSessionContexts() {
      * @param string $context The context to add to the user session.
      */
     public function addSessionContext($context) {
+        if (session_status() === PHP_SESSION_NONE) {
```

Comment:
Oops, I can probably do it in a 5.3 compatible way, but certainly up for debate...